releases/v19.1.6.md

```diff
@@ -0,0 +1,101 @@
+- Reduced write-amp by avoiding forcing files through compactions unnecessarily. [#41301][#41301]
+- Fixed a rare data corruption bug in RocksDB caused by newer Linux kernel's handling of i_generation on certain file systems. [#41393][#41393]
+- Fixed a bug causing the `cluster_logical_timestamp()` function to sometimes return incorrect results. [#41441][#41441]
+- Fixed internal errors generated during the execution of some complicated cases of correlated subqueries. [#41667][#41667]
+- Fixed a bug causing zone configuration changes on tables with existing index zone configurations to not take effect unless the `num_replicas` field was also set. [#41682][#41682]
+- Fixed bug causing zone configuration application on indexes to leak into configurations on partitions. [#41679][#41679]
+- Fixed multiple bugs relating to zone configurations and copy from parent. Previously, `COPY FROM PARENT` was ignored when using it on partitions and indexes, and when using it on a zone that had an existing value for the field that was being changed. [#41699][#41699]
+- Fixed a bug causing rapid network disconnections to lead to cluster unavailability because goroutines waited for a connection which will never be initialized to send its first heartbeat. [#42165][#42165]
+- Fixed an internal error in a rare case involving `UNION` and tuples inside `VALUES` clauses. [#42266][#42266]
+- CockroachDB now permits planning of window functions within mutation statements, and other statements that can't be distributed. [#42617][#42617]
+- Fixed a bug that would produce a spurious failure with the error message "incompatible COALESCE expressions" when adding or validating `MATCH FULL` foreign key constraints involving composite keys with columns of differing types. [#42652][#42652]
+- Fixed a case where we incorrectly determine that a query (or part of a query) which contains an `IS NULL` constraint on a unique index column returns at most one row, possibly ignoring a `LIMIT 1` clause. [#42792][#42792]
+- Now possible to transfer range leases to lagging replicas [#42764][#42764]
+- [`ALTER INDEX IF EXISTS`](../v19.1/alter-index.html) no longer fails when using an unqualified index name that does not match any existing index. Now it is a no-op. [#42839][#42839]
+- CockroachDB now prevents a number of panics from the SQL layer caused by an invalid range split. These would usually manifest with messages mentioning encoding errors ("found null on not null column" but also possibly various others). [#42860][#42860]
+- The experimental cloud storage changefeed sink previously violated some of the changefeed invariants under rare conditions. A number of these have been fixed, but some fixes require changes to the format of the filenames output by the cloud storage changefeed. This is unsuitable for inclusion in a patch release, so users of cloud storage changefeed sinks are highly encouraged to upgrade to CockroachDB 19.2 or later. [#42907][#42907]
```

Comment:
~Suggest moving to the top of the list, since it's the only one saying "hey if you use this feature you should really upgrade".